lib/commands/env.js

```diff
@@ -92,39 +97,40 @@ env.export = function(projectName, file, done) {
  *    for.
  * @param {Function} cb Callback function.
  */
-env.list = function(projectName, cb) {
+env.list = function (projectName, cb) {
   var project;
 
-  async.waterfall([
-    function(cb) {
-      env._getAndChooseProject(projectName, cb);
+  Async.waterfall([
+    function (callback) {
+      env._getAndChooseProject(projectName, callback);
     },
 
-    function(pro, cb) {
+    function (pro, callback) {
       project = pro;
-      modulus.io.print('Project ' + project.name.data + ' Environment Variables');
+      print('Project ' + project.name.data + ' Environment Variables');
 
-      addOnController.getConfigVars(project.id, cb);
+      AddOnController.getConfigVars(project.id, callback);
     },
 
-    function(addonVars, cb) {
-      for (var i = 0; i < project.envVars.length; i++) {
+    function (addonVars, callback) {
+      var i;
+      for (i = 0; i < project.envVars.length; ++i) {
         env._printEnv(project.envVars[i]);
       }
 
       if (addonVars.length > 0) {
-        modulus.io.print('\nAdd-Ons');
+        print('\nAdd-Ons');
 
-        for (var c = 0; c < addonVars.length; c++) {
-          env._printEnv(addonVars[c]);
+        for (i = 0; i < addonVars.length; ++i) {
+          env._printEnv(addonVars[i]);
         }
       }
 
-      cb(null);
+      callback(null);
     }
   ],
-  function(err) {
-    if(err) {
+  function (err) {
+    if (err) {
```

Comment:
Looks like there's a few times where these statements are one-liners and other times they are multi-line. Did you want them to be consistent?